MAINTAINERS

```diff
@@ -2,6 +2,7 @@
   "version": 1,
   "file_format": "This MAINTAINERS file format is described at https://github.com/puppetlabs/maintainers",
   "issues": "https://github.com/puppetlabs/clj-puppetdb/issues",
+  "internal_list": "https://groups.google.com/a/puppet.com/forum/?hl=en#!forum/discuss-console-ui-maintainers",
```

Comment:
I wonder if this group is appropriate for this repo. Isn't it just for `pe-console-ui`?